.changesets/feat_geal_authz_log_errors.md

```diff
@@ -0,0 +1,13 @@
+### add a flag to disable authorization error logs ([PR #4076](https://github.com/apollographql/router/pull/4076))
+
+Authorization errors can be seen as common usage of the service when filtering fields from queries depending on the client's rights, so they might not warrant error logs to be analyzed by the router operators
+
+THos logs can be disabled in the configuration:
+
+```yaml
+authorization:
+  preview_directives:
+    log_errors: true
+```
```

Comment:
I don't have strong feelings about this, but ould this be better as:

```yaml
authorization:
  preview_directives:
    logging:
      enabled: true
```
?
We seem to be settling on this `enabled` `true|false` style and I wonder if it's just about enabling `logging` for directives or specifically `errors`.

also: We should specify if enabled or not by default here and in the docs.

Reply:
cc @BrynCooke as you might have a preference in light of the telemetry recent work

Reply:
It depends if we see further logging configuration in the future.
If we think there will be more config:

```yaml
authorization:
  preview_directives:
    log:
      errors: true
```

We can always tackle this with a migration though, so happy with for now:
```yaml
authorization:
  preview_directives:
    log_errors: true
```